end-to-end-test/remote/specs/core/comparisonTab.spec.js

```diff
@@ -1,135 +1,179 @@
+const getSampleCreateGroupButton = async function() {
+    return getElementByTestHandle('sampleGroupComparisonCreateGroupButton');
+};
+const getPatientCreateGroupButton = async function() {
+    return getElementByTestHandle('patientGroupComparisonCreateGroupButton');
+};
+const dataTestPatientGroupNameSubmitButton = async function() {
+    return getElement('[data-test="patientGroupNameSubmitButton"]');
+};
+const dataTestPatientGroupNameInputField = async function() {
+    return getElement('[data-test="patientGroupNameInputField"]');
+};
+const getSampleGroupNameInputField = async function() {
+    return getElementByTestHandle('sampleGroupNameInputField');
+};
+const getSampleGroupNameSubmitButton = async function() {
+    return getElementByTestHandle('sampleGroupNameSubmitButton');
+};
 
 describe('results view comparison tab venn diagram tests', function() {
-    describe('create new group tests', function() {
-        before(function() {
-            goToUrlAndSetLocalStorage(
+    describe.only('create new group tests', function() {
+        before(async function() {
+            await asyncUtils.goToUrlAndSetLocalStorage(
                 `${CBIOPORTAL_URL}/results/comparison?Z_SCORE_THRESHOLD=2.0&cancer_study_id=coadread_tcga_pub&cancer_study_list=coadread_tcga_pub&case_set_id=coadread_tcga_pub_nonhypermut&gene_list=KRAS%20NRAS%20BRAF&gene_set_choice=user-defined-list&genetic_profile_ids_PROFILE_COPY_NUMBER_ALTERATION=coadread_tcga_pub_gistic&genetic_profile_ids_PROFILE_MUTATION_EXTENDED=coadread_tcga_pub_mutations`
             );
-            $('div[data-test="ComparisonPageOverlapTabDiv"]').waitForExist({
-                timeout: 20000,
-            });
+            await asyncUtils.getElementByTestHandle(
+                'ComparisonPageOverlapTabDiv',
+                {
+                    timeout: 20000,
+                }
+            );
         });
 
-        it('create group button disabled as default', function() {
-            assert.equal($(SampleCreateGroupButton).isEnabled(), false);
-            assert.equal($(PatientCreateGroupButton).isEnabled(), false);
+        it('create group button disabled as default', async function() {
+            assert.equal(
+                (await getSampleCreateGroupButton()).isEnabled(),
+                false
+            );
+            assert.equal(
+                (await getPatientCreateGroupButton()).isEnabled(),
+                false
+            );
         });
 
-        it('select from sample venn diagram', function() {
-            jsApiClick('rect[data-test="sample0VennRegion"]');
+        it('select from sample venn diagram', async function() {
+            await jsApiClick('rect[data-test="sample0VennRegion"]');
             browser.pause(100);
-            assert.equal($(SampleCreateGroupButton).isEnabled(), true);
-            assert.equal($(PatientCreateGroupButton).isEnabled(), false);
+            assert.equal(
```

Comment:
fyi, browser.pause needs to be awaited as well.  in async mode, literally ever command is async so needs to be awaited.